README.md

```diff
@@ -1,9 +1,12 @@
 # sanic-useragent
-Add `user_agent` to request for Sanic.
+Add `user_agent` to request `ctx` for Sanic.
 
 ## Installation
 
-`pip install sanic-useragent`
+```bash
+pip install git+https://github.com/pawelkoston/sanic-useragent.git
```

Comment:
we still want install it via pypi

Reply:
I am sorry, reverted to the original README.md